### PR TITLE
Add blog post justifying usage of python

### PR DESCRIPTION
**Pull request by RaphHaddad**

### CODE REVIEW

File: content/post/python-to-automate-azure-deployments.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+The Azure CLI can definitely be run on Windows, however, due to the different shells available on Windows (PowerShell, Command Prompt) a script may experience negative unintended consequences. Specifically, Windows shells such as PowerShell and Command Prompt deal with quotes (and escaping them) differently. [See this documentation from Microsoft](https://docs.microsoft.com/en-us/cli/azure/use-cli-effectively#quoting-issues) that goes into this issue in more detail.
+
+Python’s design makes it easy to split code in modules, files, and classes. This helps keep related code close to each other and aides in dependancy management of modules. Using the Azure CLI within Python is simple by either creating a wrapper using the Python libraries argparse and json, or using the [existing pip library by Microsoft](https://github.com/Azure/azure-cli). In addition, the Azure CLI itself is written Python, so if a bug is found, context switching into the Azure CLI code won’t be a strenuous affair.
```
Comment (glav) [inline content/post/python-to-automate-azure-deployments.md:45]: Azure CLI itself is written in Python

File: content/post/python-to-automate-azure-deployments.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Python’s design makes it easy to split code in modules, files, and classes. This helps keep related code close to each other and aides in dependancy management of modules. Using the Azure CLI within Python is simple by either creating a wrapper using the Python libraries argparse and json, or using the [existing pip library by Microsoft](https://github.com/Azure/azure-cli). In addition, the Azure CLI itself is written Python, so if a bug is found, context switching into the Azure CLI code won’t be a strenuous affair.  
+
+## Integration with Infrastructure Template Technologies
+
+Due to the ease of modularisation explained above Python also makes it easy to integrate with infrastructure template technologies such as Terraform, Bicep, or ARM Templates.  Python’s libraries that integrate with the host operating system’s shell means that CLI tools from a variety of providers can be used by building out the necessary wrappers and modules over them.
```
Comment (glav) [inline content/post/python-to-automate-azure-deployments.md:49]: Due to the ease of modularisation explained above, Python also makes it .....

File: content/post/python-to-automate-azure-deployments.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Integration with Infrastructure Template Technologies
+
+Due to the ease of modularisation explained above Python also makes it easy to integrate with infrastructure template technologies such as Terraform, Bicep, or ARM Templates.  Python’s libraries that integrate with the host operating system’s shell means that CLI tools from a variety of providers can be used by building out the necessary wrappers and modules over them. 
+
+## Readability and Conciseness
+
+Many scripting languages run on Linux: Perl, Ruby, and Bash (amongst many). As a consequence, choosing Python over other languages may seem inconsequential, however, Python is a lot more readable and easier to learn than many other languages. Many introduction to coding courses are now using Python to teach coding (both in schools and in higher education institutes). A testament to the simplicity of the language.
```
Comment (glav) [inline content/post/python-to-automate-azure-deployments.md:53]: "Python is a lot more readable and easier to learn than many other languages" - opinion alert :-) Worth stating "in my opinion"?

**Review by glav — CHANGES_REQUESTED** [review]

File: content/post/python-to-automate-azure-deployments.md (hunk 1 above)
Comment (RaphHaddad) [inline content/post/python-to-automate-azure-deployments.md:45]:
```suggestion
Python’s design makes it easy to split code in modules, files, and classes. This helps keep related code close to each other and aides in dependancy management of modules. Using the Azure CLI within Python is simple by either creating a wrapper using the Python libraries argparse and json, or using the [existing pip library by Microsoft](https://github.com/Azure/azure-cli). In addition, the Azure CLI itself is written in Python, so if a bug is found, context switching into the Azure CLI code won’t be a strenuous affair.  
```

File: content/post/python-to-automate-azure-deployments.md (hunk 2 above)
Comment (RaphHaddad) [inline content/post/python-to-automate-azure-deployments.md:49]:
```suggestion
Due to the ease of modularisation explained above, Python also makes it easy to integrate with infrastructure template technologies such as Terraform, Bicep, or ARM Templates.  Python’s libraries that integrate with the host operating system’s shell means that CLI tools from a variety of providers can be used by building out the necessary wrappers and modules over them. 
```

File: content/post/python-to-automate-azure-deployments.md (hunk 3 above)
Comment (RaphHaddad) [inline content/post/python-to-automate-azure-deployments.md:53]:
```suggestion
Many scripting languages run on Linux: Perl, Ruby, and Bash (amongst many). As a consequence, choosing Python over other languages may seem inconsequential, however in my opinion, Python is a lot more readable and easier to learn than many other languages. Many introduction to coding courses are now using Python to teach coding (both in schools and in higher education institutes). A testament to the simplicity of the language. 
```

File: content/post/python-to-automate-azure-deployments.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+---
+title: 'Python to Automate Azure Deployments'
+date: 2021-03-12T00:00:00.001+11:00
```
Comment (RaphHaddad) [inline content/post/python-to-automate-azure-deployments.md:3]:
```suggestion
date: 2021-03-16T00:00:00.001+11:00
```

**Pull request merged**